src/main/com/aca/homework/week9/calculator/SimpleCalculator.java

```diff
@@ -0,0 +1,51 @@
+package com.aca.homework.week9.calculator;
+
+public class SimpleCalculator {
+    private final StringSupplier supplier;
+
+    public SimpleCalculator(StringSupplier supplier) {
+        this.supplier = supplier;
+    }
+
+    public String askAndCalculate() {
+        System.out.println("Enter an expression in a format [digit][operation][digit]");
+        String expression = this.supplier.get();
+
+        if (!isValidExpression(expression)) {
+            System.out.println("error");
+            return "error";
+        }
+
+        int firstDigit = getDigitFromChar(expression.charAt(0));
+        int secondDigit = getDigitFromChar(expression.charAt(2));
+        char operation = expression.charAt(1);
+        switch (operation) {
```

Comment:
the method is too big. Let's create an Operation enum class and implement calculate method for each enum value

Reply:
also create a static method in enum class that will take a char as an argument and will return enum value